parser/src/lexer.rs

```diff
@@ -465,6 +466,20 @@ where
         }
     }
 
+    /// abandon a comment
```

Comment:
```suggestion
    /// Discard comment if full-lexer is not enabled.
```

Reply:
The function return type is different. Maybe this part can be `self.read_until_line_end()` later